mysql-test/suite/rpl/t/rpl_bug-lp-1667307.test

```diff
@@ -0,0 +1,51 @@
+#
+# In the case if the bug is not fixed the test will crash on assert
+# in Rows_query_log_event::do_apply_event().
+#
+--source include/have_debug.inc
```

Comment:
I see no reason not to run this on release build as well, this a regression test case after all